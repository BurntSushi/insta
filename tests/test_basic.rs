extern crate insta;

use insta::{assert_debug_snapshot_matches};

#[test]
fn test_vector() {
    assert_debug_snapshot_matches!("vector", vec![1, 2, 3]);
}

#[test]
#[cfg(feature = "serialization")]
fn test_serialized_vector() {
    use insta::assert_serialized_snapshot_matches;
    assert_serialized_snapshot_matches!("serialized_vector", vec![1, 2, 3]);
}
