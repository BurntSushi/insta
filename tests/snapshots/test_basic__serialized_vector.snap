---
created: "2019-01-22T21:33:27.842166+00:00"
creator: insta@0.4.1
expression: "vec![1 , 2 , 3]"
source: tests/test_basic.rs
---
- 1
- 2
- 3
